samples/NativeLibrary/README.md

```diff
@@ -23,11 +22,80 @@ The above command will drop a static library (Windows `.lib`, OSX/Linux `.a`) in
 ## Building shared libraries
 
 ```bash
+
 > dotnet publish /p:NativeLib=Shared -r <RID> -c <Configuration>
+
 ```
 
 The above command will drop a shared library (Windows `.dll`, OSX `.dylib`, Linux `.so`) in `./bin/[configuration]/netstandard2.0/[RID]/publish/` folder and will have the same name as the folder in which your source file is present.
 
+### Loading shared libraries from C and importing methods
+For reference, you can read the two C files located in the c_source folder.
+The first thing you'll have to do in order to have a proper "loader" that loads your shared library is to add these directives
+```c
+    #ifdef _WIN32
+    
+    #include "windows.h"
+    
+    #else
+    
+    #include "dlfcn.h"
+    
+    #define __stdcall
```

Comment:
Maybe you can strip `__stdcall` from here and other places - since you have stripped it from the `.cs` file. It is very Windows x86 specific and you can deal with it by using the right command line options for the compiler on Windows, no need to spread it through everywhere.

Reply:
So the __stdCall isn't strictly required on windows to declare a function pointer ?

Reply:
Correct.